src/main/webapp/i18n/en/quizExercise.json

```diff
@@ -22,6 +22,8 @@
                     "value": "Multiple Choice: {{ mcCount }}, Drag And Drop: {{ dndCount }}, Short Answer: {{ shortCount }}"
                 }
             },
+            "create": "Create Quiz",
+            "import": "Import Quiz",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys